doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+
+### First step id=xs1st
+
+The first step is to generate a 1041-group `rzmflx`, which is the flux solution file of a TWODANT [!citep](TWODANT) SN calculation. It contains zone-wise Legendre moments of the angular flux solution up to a specified order. TWODANT is called inside MC2-3 when its executable path is specified by the `c_twodantexe` input parameter in the `library` block of an MC2-3 input with `l_twodant=T`. If TWODANT is not available, PARTISN [!citep](PARTISN) can be used alternatively since it creates a `rzmflx` file. `c_twodant_group=BG` means that the TWODANT transport calculation is performed using a group structure specified in `c_group_structure`. `c_geometry_type=mixture` means that homogeneous materials cooresponding to axial regions are used to generate macroscopic cross sections (in the 1041-group structure in this example) to be used in the TWODANT transport calculation. 
+
+!listing lfr/heterogeneous_single_assembly_3D/cross_section/Step1/LFR_InnerFuel_HOM_1041g_TWODANT.mcc3.sh start=\$library end=\$material include-end=False
+
+Homogenized compositions representing different axial regions are specified in the `material` block. In this step, temperature does not need to be very accurate.
+
+!listing lfr/heterogeneous_single_assembly_3D/cross_section/Step1/LFR_InnerFuel_HOM_1041g_TWODANT.mcc3.sh start=\$material end=/ include-end=True
+
+The `twodant` block is to generate a TWODANT input internally for calling TWODANT. `niso` and `ngroup` are filled by MC2-3, `mt` and `nzone` are the number of materials and coarse meshes, `im` and `jm` are the number of coarse meshes in the x and y directions, `it` and `jt` are the total number of fine meshes in the x and y directions, `igeom=6` means x-y geometry, `isn` is the Sn order, and `maxlcm` and `maxscm` are the maximum memory size of central small core memory and peripheral large core memory. `xmesh` and `ymesh` are x and y coordinates of coarse meshes, `xints` and `yints` are the number of fine meshes in each coarse x and y meshes, and `zones` are composition numbers in the `material` block assigned to each x and y coarse mesh.
+
+Input parameters from `isct` to `rmflux` are solver options. Important ones are `isct`: Legendre order of scattering, `ievt=1`: calculation type=k-eff, `ibl, ibr, ibt, ibb`: boundary conditions for left, right, top and bottom (0/1/2/3=vacuum/reflective/periodic(only for top and bottom)/white), `epsi`: convergence precision, `oitm`: maximum number of outer iterations, `iitm, iitl`: maximum number of inner iterations per group at the first outer iteration and near fission source convergence.
```

Comment:
I think here you need to mention that left/right boundary are reflecting, thus this 2D calculation is a pseudo 1D problem. The reason of doing this is that TWODANT only supports 2D calculations. I was confused on this when I saw 1D slab model in Figure 6 because I was not paying attention on the boundary condition here.

Reply:
I added `epso`.
```
epsi: inner iteration convergence precision (default=0.0001), epso: outer iteration convergence precision (default=epsi),
```
To avoid confusion, I added the following sentence in the `First step`.
```
In this problem, a pseudo one-dimensional (1D) slab problem is solved using TWODANT by assigning the reflective boundary condition on the left and right boundaries in the x-y geometry. 
```